### ISSUE TITLE
Connection closed

### ISSUE THREAD

**Issue by absentvs** [Sunoo/homebridge-smtp-motion#86]
Hello! Thank you for your job!

When motion in Homebridge debug log see this messages:
_[28/12/2020, 11:23:14] [SMTP Motion] [SMTP Server] Connection closed to 192.168.1.152
[28/12/2020, 11:23:14] [SMTP Motion] [SMTP Server] Reverse resolve for 192.168.1.152: getHostByAddr ENOTFOUND 192.168.1.152_

Where 192.168.1.152 - it's a IP of a cam (IPC Dahua).

Reaction is visible in log. But doesn't work for trigger.

**Comment by Sunoo** [comment]
Huh, I’ve not seen that before. Is your network setup doing anything complex with VLANs/firewalls/etc? Are you able to ping the camera’s IP address from whatever device you’re running Homebridge on?

**Comment by absentvs** [comment]
I use a regular network without additional settings.
Ping results in Raspberry with Homebridge on:

pi@homebridge:/ $ telnet 192.168.1.152 80
Trying 192.168.1.151...
Connected to 192.168.1.151.

pi@homebridge:/ $ ping 192.168.1.152
PING 192.168.1.152 (192.168.1.152) 56(84) bytes of data.
64 bytes from 192.168.1.152: icmp_seq=1 ttl=64 time=0.590 ms
64 bytes from 192.168.1.152: icmp_seq=2 ttl=64 time=0.497 ms
...

What ports need to ping additionally?

**Comment by absentvs** [comment]
Perhaps, in the case of Dahua, there is a problem with authorizing the camera connection via SMTP. During the SMTP signaling configuration phase, verification is performed and an error occurs for nonexistent parameters and ports.
I found a similar issue for Reolink cameras in issues here.

In my case, I decided to use another way through the homebridge-dahua-alerts plugin.
The question is no longer relevant. Thank you.
